views/case-list.njk

```diff
@@ -1,7 +1,73 @@
+{% set tableData = {
+  head: [
+    { text: "Defendant" },
+    { text: "Probation record" },
```

Comment:
Alistair has changed the content for the "Probation record" column, it's now "Probation status"

Reply:
Ack; sorry I'll change that.